packages/wallet-connectors/src/WalletConnect.ts

```diff
@@ -119,6 +120,31 @@ function serializeUpdateContractMessage(
     }
 }
 
+/**
+ * Convert schema into the object format expected by the Mobile crypto library.
+ * @param schema The schema object.
+ */
+function serializeSchema(schema: Schema | undefined) {
```

Comment:
```suggestion
function convertSchemaFormat(schema: Schema | undefined) {
```
At least I wouldn't say this function serializes the schema. :thinking:

Reply:
done